iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/BaseRoutePreviewStatus.swift

```diff
@@ -118,15 +118,16 @@ final class BaseRoutePreviewStatus: SolidTouchView {
     if MWMRouter.hasRouteAltitude() {
       heightBox.isHidden = false
       MWMRouter.routeAltitudeImage(for: heightProfileImage.frame.size,
-                                   completion: { image, elevation in
+                                   completion: { image, totalAscent, totalDescent in
                                      self.heightProfileImage.image = image
-                                     guard let elevation = elevation else { return }
+                                     guard let totalAscent = totalAscent else { return }
+									 guard let totalDescent = totalDescent else { return }
```

Comment:
1. Tabs and spaces are also mixed. Please clean up.
2. Is guard really needed here? Why not a simple `if`?

Reply:
- Fixed idents
- Replaced 2 guard by if
d80c3a0